br/pkg/restore/client.go

```diff
@@ -494,6 +494,32 @@ func (rc *Client) GetTS(ctx context.Context) (uint64, error) {
 	return restoreTS, nil
 }
 
+// GetTSWithRetry gets a new timestamp with retry from PD.
+func (rc *Client) GetTSWithRetry(ctx context.Context) (uint64, error) {
+	var startTS uint64
+	var getTSErr error = nil
+	var retry uint = 0
```

Comment:
Perhaps we can emit zero values and merge those `var`s:
```suggestion
	var (
		startTS uint64
		getTSErr error
		retry uint
	)
```